### PR TITLE
IF-4274 Support connection Azure

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: fic/eri/v1/port_to_azure_microsoft_connections/testing/fixtures.go [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+                }
+            },
+            "destination": {
+                "interconnect": "Tokyo-1",
+                "qosType": "guarantee",
+                "serviceKey": "6191af11-82f9-4c15-9894-9a69c8f8628a",
+                "sharedKey": "a8268f8c96a9",
+                "advertisedPublicPrefixes": [
+                    "100.100.1.1/32",
+			        "100.100.1.2/32"
```
Comment (keiichi-hikita) [inline fic/eri/v1/port_to_azure_microsoft_connections/testing/fixtures.go]: duplicate indent?

**Review by keiichi-hikita — CHANGES_REQUESTED** [review]

File: fic/eri/v1/port_to_azure_microsoft_connections/testing/fixtures.go (hunk 1 above)
Comment (ghost) [reply fic/eri/v1/port_to_azure_microsoft_connections/testing/fixtures.go]: fixed!

**Review by keiichi-hikita — APPROVED** [review]
LGTM o/

**Review by shimisho — APPROVED** [review]
LGTM.

**Pull request merged**